ci-operator/step-registry/azure/provision/vnet-hypershift/azure-provision-vnet-hypershift-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "azure/provision/vnet-hypershift/azure-provision-vnet-hypershift-ref.yaml",
```

Comment:
Hmm, I did not see the change here?

Reply:
Oh, I misunderstood. The `vnet` directory is already taken so I think it has to be `vnet-hypershift`.

Reply:
You can place a sub-folder in `vnet` directory, of course, if you like the the folder on the same level as `vnet` folder, I do not have objections either.

Reply:
I'd like to keep the current implementation but thank you for the suggestion.